command/repo.go

```diff
@@ -2,19 +2,27 @@ package command
 
 import (
 	"fmt"
+	"net/url"
 	"os"
 	"strings"
 
+	"github.com/cli/cli/api"
 	"github.com/cli/cli/git"
 	"github.com/cli/cli/internal/ghrepo"
 	"github.com/cli/cli/utils"
+
+	"github.com/AlecAivazis/survey/v2"
 	"github.com/spf13/cobra"
 )
 
 func init() {
 	RootCmd.AddCommand(repoCmd)
 	repoCmd.AddCommand(repoCloneCmd)
 	repoCmd.AddCommand(repoViewCmd)
+	repoCmd.AddCommand(repoForkCmd)
+
+	repoForkCmd.Flags().BoolP("yes", "y", false, "Run non-interactively, saying yes to prompts")
+	repoForkCmd.Flags().BoolP("no", "n", false, "Run non-interactively, saying no to prompts")
```

Comment:
Idea: maybe we can be more specific with flag naming here and have the flag betray the specific operation that it applies to? E.g. `repo fork --clone` or `repo fork --clone=false`

I feel like `--yes` and `--no` could prove hard to maintain over time if it turns out they mean different things in different commands, and in case those commands have more than one boolean prompt

Reply:
just having clone doesn't help the remote adding case. i based y/n on apt-get and i like how it's easy to reason about in different contexts.

that said, i could be ok with --add-remote and --clone. i know @ampinsk pointed this out too; what do you think?

Reply:
Yeah, I like the idea of us being explicit with our flag language like this. I'm into `--add-remote` and `--clone` with the options to set them to `false` as well

Reply:
:+1: I'll switch to `--add-remote` / `-a` and `--clone` / `-c`

Reply:
i'm not super thrilled about this but here's what I came up:
```
Flags:
  -c, --clone string    true: clone fork. false: never clone fork (default "prompt")
  -r, --remote string   true: add remote for fork. false: never add remote fork (default "prompt")
```

Reply:
I tried tweaking a bit:
```
Flags:
  -c, --clone string    Clone fork. Pass false to not clone fork. No flag will prompt.
  -r, --remote string   Add remote for fork. Pass false to not add remote fork. No flag will prompt.
```

Reply:
distinguishing between "user specified flag but passed nothing" and "user did not specify flag" is weirdly unintuitive in our codebase; I got frustrated and did the explicit true/false thing. I can take another swing at it though.

Reply:
ah I immediately figured out how to do your suggestion, @ampinsk :woman_facepalming: so now a bare `--clone` or `--remote` implies `true`.

Reply:
thanks nate!!